File: Basic-Programs/Program15.java
import java.util.Scanner;


class Program15
{
    public static void main(String A[])
    {
        float fPercentage = 0.0f;
        
        Scanner Sc = new Scanner(System.in);
        
        System.out.println("Enter Your Percentage : ");
        fPercentage = Sc.nextFloat();
        
        if(fPercentage >= 35)
        {
            System.out.println("You Are Pass ");
        }
        else
        {
            System.out.println("You Are Fail ");
        }
        
    }
}

File: Basic-Programs/Program15.c
#include<stdio.h>

int main()
{
    float fValue = 0.0f;
    
    printf("Enter The Percentage : \n");
    scanf("%f",&fValue);
    
    if(fValue >= 40.00f)
    {
        printf("You Are Pass \n");
    }
    else
    {
        printf("Your Area Fail \n");
    }
    
    return 0;
}